.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,23 +10,64 @@ assignees: ''
 
 The release number or commit hash of the version you're using.
 
-## Operating system & version
+## \[Optional\] Relevant system infromation
 
-Ex: Windows 10, Ubuntu 18.04, iOS 14.
+If you are reporting a bug about:
+
+- difficulties getting Bevy to build or run on your machine
+- unusual rendering bugs
+- unusual input bugs
+- hardware specific problems
+
+These issues are often specific to hardware or OS configuration and can be challenging to reproduce.
+As a result, please consider including information about:
+
+- the Rust version you're using (you can get this by running `cargo --version`)
+  - Bevy has a policy of relying on the "latest stable release" of Rust
+  - Nightly should generally work, but there are sometimes regressions: please let us know!
+- the operating system, including its version
+  - e.g. Windows 10, Ubuntu 18.04, iOS 14.
+- relevant hardware
+  - if your bug is rendering-related, copy the adapter info that appears when you run Bevy
+  - e.g. `AdapterInfo { name: "NVIDIA GeForce RTX 2070", vendor: 4318, device: 7938, device_type: DiscreteGpu, backend: Vulkan }` or XBox series X controller
+
+You should also consider testing the examples of our upstream dependencies to help isolate the issue:
+
+- [`wgpu`](https://github.com/gfx-rs/wgpu) for rendering problems
+- [`winit`](https://github.com/rust-windowing/winit) for input and window management
+- [`gilrs`](https://docs.rs/gilrs/latest/gilrs/) for gamepad inputs
 
 ## What you did
 
-The steps you took to uncover this bug. Please list full reproduction steps if
-feasible.
+The steps you took to uncover this bug.
+Please provide a runnable code snippet or link to an example that demonstrates the problem if you can.
```

Comment:
This should feel more "optional". The current wording is a bit too strong. I suspect it may lead some users to feel like they have to muster up some code example to report a bug to us.

```suggestion
If you can, it would be helpful to provide a code snippet (or link to one) that demonstrates the problem.
```

Reply:
in your new commit you already did better than what i suggested